karton/core/main.py

```diff
@@ -42,30 +154,82 @@ def process(self, task):
 
 
 def main() -> None:
+
     parser = argparse.ArgumentParser(description="Your red pill to the karton-verse")
-    parser.add_argument("--list", action="store_true", help="List active karton binds")
+    parser.add_argument("--version", action="version", version=__version__)
+    parser.add_argument("-c", "--config-file", help="Alternative configuration path")
     parser.add_argument(
-        "--delete", action="store", help="Delete persistent karton bind"
+        "-v", "--verbose", action="store_true", help="More verbose log output"
     )
-    parser.add_argument("--version", action="version", version=__version__)
-    parser.add_argument("--config-file", help="Alternative configuration path")
-    args = parser.parse_args()
 
-    config = Config(args.config_file)
+    subparsers = parser.add_subparsers(dest="command", help="sub-command help")
```

Comment:
maybe some `required=True` would help? missing param will print help in this case